xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -138,8 +154,8 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.TimeSpan</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+         <summary>Gets or sets the timespan to wait before the connection establishing times out.</summary>
```

Comment:
I'd remove the space added to avoid CI touching this later

```suggestion
        <summary>Gets or sets the timespan to wait before the connection establishing times out.</summary>
```